spec/models/landing_page/block/component_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe LandingPage::Block::Component do
+  it_behaves_like "it is a landing-page block"
+
+  let(:blocks_hash) do
+    { "type" => "component",
+      "component_name" => "big_number",
+      "number" => 123,
+      "label" => "Number of things" }
+  end
+  let(:subject) { described_class.new(blocks_hash, build(:landing_page)) }
+
+  describe "allowed list of components" do
```

Comment:
Should this be `describe "#component_name"` then have the following two nested contexts/tests:

* `context "when the component_name is in the allowed list"`
* * `it "sets the  components name"`
* `context "when the component_name is not in the allowed list"`
* * `it "raises an error"`